src/Aspire.Dashboard/Components/Pages/NotFound.razor.css

```diff
@@ -0,0 +1,17 @@
+.error-stack {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: calc(var(--design-unit) * 10px);
+}
+
+.error-code {
+    --error-code-font-size: 90px;
```

Comment:
Is there a reason this size is not in terms of any existing variable>

Reply:
None of the existing variables go anywhere near that high. +6 is the largest and it was only 40px. Since it was only for one place on one page I didn't think it made sense to try to concoct a `--type-ramp-plus-25-*` recipe or whatever it'd be for that large.